tests/test_omegaconf.py

```diff
@@ -117,6 +123,13 @@ def test_is_missing_resets() -> None:
     assert OmegaConf.is_missing(cfg, "list")
 
 
+def test_dereference_interpolation_to_missing() -> None:
+    cfg = OmegaConf.create({"x": "${y}", "y": "???"})
+    x_node = cfg._get_node("x")
+    assert isinstance(x_node, Node)
+    assert x_node._dereference_node() is None
+
+
```

Comment:
Does accessing`cfg.x` raises an MandatoryMissing exception?

Reply:
Oh yes, it would be quite bad otherwise. I added it to the test in 9247d0b to make it clear.